app/Tasks/Deploy.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace App\Tasks;
+
+use App\Builder\Builder;
+use App\Environment;
+use Illuminate\Support\Facades\File;
+use Illuminate\Support\Facades\Http;
+use Illuminate\Support\Str;
+
+class Deploy extends Task
+{
+    const URL = 'http://fwd.tools';
```

Comment:
this should be https, and also possibly come from an ENV variable por easier changing for testing.